source/features/lines-changed-per-commit.tsx

```diff
@@ -0,0 +1,52 @@
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL} from '../libs/utils';
+import React from 'dom-chef';
+import elementReady from 'element-ready';
+
+const getCommitChanges = async (commit: string): Promise<number[]> => {
```

Comment:
Can you add `cache.function`?

Also this type would be more correct because there are only 2 values: 
```suggestion
const getCommitChanges = async (commit: string): Promise<[number, number]> => {
```